src/compressor.cpp

```diff
@@ -22,7 +22,7 @@ static bool IsToKeyID(const CScript& script, CKeyID &hash)
     if (script.size() == 25 && script[0] == OP_DUP && script[1] == OP_HASH160
                             && script[2] == 20 && script[23] == OP_EQUALVERIFY
                             && script[24] == OP_CHECKSIG) {
-        memcpy(&hash, &script[3], 20);
+        memcpy(hash.begin(), &script[3], 20);
```

Comment:
Can we instead do `CKeyID.GetRaw` that points to the raw hash and just use the address of that? 

Since memcpy shouldn't be relying on addresses returned by iterators. Iterator address can be different from the raw hashes.

Reply:
Sure, I hear that - however `begin`, `end` is usually associated with iter interfaces and hence the nitpick to avoid a misrepresented convention.